integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/management/CarbonLoadDataCommand.scala

```diff
@@ -837,7 +836,6 @@ case class CarbonLoadDataCommand(
           query = logicalPlan,
           overwrite = false,
           ifPartitionNotExists = false)
-      sparkSession.sparkContext.setLocalProperty(EXECUTION_ID_KEY, null)
```

Comment:
Old version (spark 2.2 and below , some carbon functionlities will not work if you don't set to null) 
as this change done in 2.3 spark.
https://github.com/apache/spark/commit/9f6b3e65ccfa0daec31b58c5a6386b3a890c2149

so don't set null , `if (SparkUtil.isSparkVersionXandAbove("2.3")`
else we need to set null.

Reply:
@ajantha-bhat Handled the comment.